flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/MultipleProgramsTestBase.java

```diff
@@ -80,29 +80,36 @@
 	
 	protected final TestExecutionMode mode;
 
-	
+	private TestEnvironment testEnvironment;
+
+	private CollectionTestEnvironment collectionTestEnvironment;
+
 	public MultipleProgramsTestBase(TestExecutionMode mode) {
 		this.mode = mode;
-		
+
 		switch(mode){
 			case CLUSTER:
-				new TestEnvironment(cluster, 4).setAsContext();
+				testEnvironment = new TestEnvironment(cluster, 4);
```

Comment:
@zentol I just found that I can not move these codes into \@Before method, because it will not work with JUnit Parameterized. I proposal to keep these codes in constructor. What do you think?